tests/lean/run/structuralIssue.lean

```diff
@@ -1,15 +1,3 @@
-
-
-def f (xs : List Nat) : Nat :=
-match xs with
-| [] => 0
-| x::xs =>
-  match xs with
-  | []  => 0
-  -- This examples fails if we write the following alternative as
-  -- | _ => f xs + 1
-  | xs => f xs + 1
-
```

Comment:
What did you have to delete this test?

Reply:
I moved it, because it's not an issue anymore, in fact. It's working now, which is why I moved it to the other test file.

Reply:
Let's keep it here. The PR is about structural recursion for inductive predicates, and it is not really related to this issue. It is also not the one that fixed the issue in the comment. We can have a separate commit removing the comment above.

Reply:
ok, will revert the file.